packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-email/main/go/integrationsteps.md

```diff
@@ -15,66 +14,47 @@ if err != nil {
 }
 ```
 
-### 2: Capture credentials with the sign-in page
+Display a sign-in page that captures the username and password.
 
-Build a sign-in page that captures the username and password.
+<div class="half border">
 
-<div class="half">
-
-![Displays a basic sign-in page in a Golang application](/img/oie-embedded-sdk/oie-embedded-sdk-go-use-case-basic-sign-on-page.png)
+![A sign-in page with username and password fields and a Sign in button](/img/oie-embedded-sdk/wireframes/pwd-optional-sign-up-link-sign-in-page-g1r7.png)
 
 </div>
 
-During page load, call the `Client` object's `InitLogin` method. This method returns an object of type
-`LoginResponse` that is used to initiate the sign-in process with Okta. The object
-also contains a list of available social Identity Providers (IdPs) that is discussed in more detail in the
-[Sign in with Facebook](/docs/guides/oie-embedded-sdk-use-case-sign-in-soc-idp/go/main/)
-use case.
+During page load, call `Client.InitLogin()`. This returns an object of type `LoginResponse` that is used to initiate the sign-in process with Okta.
```

Comment:
```suggestion
During page load, call `Client.InitLogin()`. This method returns an object of type `LoginResponse` used to initiate the sign-in process with Okta.
```

Reply:
Why add the word "method" back in? We prefer to remove it if at all possible.